lxd/permissions.go

```diff
@@ -222,8 +224,32 @@ func getPermissions(d *Daemon, r *http.Request) response.Response {
 	}
 
 	if recursion == "1" {
+		slices.SortFunc(apiPermissionInfos, func(a, b api.PermissionInfo) int {
+			if result := cmp.Compare(a.EntityType, b.EntityType); result != 0 {
+				return result
+			}
+
+			if result := cmp.Compare(a.EntityReference, b.EntityReference); result != 0 {
+				return result
+			}
+
+			return cmp.Compare(a.EntityReference, b.EntityReference)
+		})
+
 		return response.SyncResponse(true, apiPermissionInfos)
 	}
 
+	slices.SortFunc(apiPermissions, func(a, b api.Permission) int {
```

Comment:
Would you be able to assign this function to a local variable instead of writing it out twice?

Reply:
I missed that; this is alright then :+1: